test/components/connect.spec.js

```diff
@@ -1469,17 +1468,17 @@ describe('React', () => {
       const target = TestUtils.findRenderedComponentWithType(tree, Passthrough)
       const wrapper = TestUtils.findRenderedComponentWithType(tree, StatefulWrapper)
 
-      expect(mapStateSpy.calls.length).toBe(2)
-      expect(mapDispatchSpy.calls.length).toBe(2)
+      expect(mapStateSpy.calls.length).toBe(1)
```

Comment:
I think the test is wrong and that the number of calls should be 1 instead of 2. I think there's a bug in current implementation that does an additional non-required call